src/components/game-setup/stepper-components/GameType.js

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { Button, Container } from '@mui/material';
+import './setup.css';
+
+function GameType() {
+  return (
+    <Container className="gameTypeBox">
+      <h1>Choose Your Game Type</h1>
+      <section className="btnContainer">
+        <div className="leftCol">
+          <Button className="guidedButton">Guided</Button>
```

Comment:
Don't be afraid to use material in order to style your components.

[Examples here](https://mui.com/material-ui/react-button/) - click the `<>` button under the examples to show the code.
[Full API here](https://mui.com/material-ui/api/button/) - Shows you what properties you can use with button.